### ISSUE TITLE
Organisation URLs use underscores, not dashes

### ISSUE THREAD

**Issue by paulrobertlloyd** [x-govuk/govuk-services-list#846]
👀

**Comment by frankieroberto** [comment]
I'm pretty relaxed about this. I know that Google used to advise against it for SEO reasons, but is that still a big deal? 🤷‍♂️

**Comment by paulrobertlloyd** [comment]
Well, beside what Google may want, the rest of the site (and GOV.UK) uses `-`, so its a bit out of character (pun intended).

**Comment by frankieroberto** [comment]
Yeah, fair. I always had a personal preference for underscores (to help disambiguate `man-eating_shark` from `man_eating_shark`) but I think I lost that one... 😄 

Changing it would involve breaking URLs, so ideally we’d add a redirect rule to convert all underscores to hyphens.
